external/wakaama/examples/bootstrap_server/bootstrap_info.h

```diff
@@ -16,7 +16,7 @@
  *******************************************************************************/
 
 #include <stdio.h>
-#include <protocols/libl2mwm.h>
+#include <protocols/liblwm2m.h>
```

Comment:
This commit is "os:Kconfig Add new menuconfigs(Runtime, Device Manage)". You can't add these changes in here.